K8s/k8s-deploy.sh

```diff
@@ -32,6 +32,13 @@ kubectl apply -f redis-service.yaml
 kubectl apply -f kafka-service.yaml
 kubectl apply -f api-service.yaml
 kubectl apply -f s3api-service.yaml
+kubectl apply -f s3-service.yaml
```

Comment:
Instead of adding so much of appy statement, you can put all the yaml in a directory and just put 1 statement as `kubectl apply -f directoryName/`

Reply:
Yes there is an issue opened for this.. As it has no impact.. will be taken up as a backlog.

Reply:
Same opinion here